deployment/single-server/app.yml

```
@@ -0,0 +1,106 @@
+---
+
+# Invoke using:
+#
+#     ansible-playbook app.yml --become --ask-become-pass
+#
+
+- name: Catalogi Importer
+  hosts: all
+
+  collections:
+    - maykinmedia.commonground
+
+  vars:
+    # Application
+    django_app_docker_name_prefix: objecttypes
+    django_app_docker_domain: TODO
+    django_app_docker_secret_key: 'TODO'
+    django_app_docker_package_name: objecttypes
+    django_app_docker_https: false
+
+    ## Application database settings
+    django_app_docker_db_host: TODO
+    django_app_docker_db_name: objecttypes
+    django_app_docker_db_username: objecttypes
+    django_app_docker_db_password: TODO
+
+    ## Application docker container
+    django_app_docker_version: latest
+    django_app_docker_image_name: maykinmedia/objecttypes-api
+    django_app_docker_replicas: 1
+    django_app_docker_port_start: 12000
```

```suggestion
    django_app_docker_port_start: 12001
```

I know it's pretty arbitrary but for testing purposes let's not use the same docker port for both the objects api and objecttypes api.

---

Sure!